lib/brainstem/api_docs/formatters/open_api_specification/helper.rb

```diff
@@ -0,0 +1,66 @@
+module Brainstem
+  module ApiDocs
+    module Formatters
+      module OpenApiSpecification
+        module Helper
+          def presenter_title(presenter)
+            presenter.contextual_documentation(:title).presence ||
+                presenter.target_class.underscore.singularize.titleize.strip
+          end
+
+          def format_http_method(endpoint)
+            endpoint.http_methods.first.downcase
+          end
+
+          def format_tag_name(name)
+            return name if name.blank?
+
+            name.underscore.titleize.strip
+          end
+
+          def format_description(description)
+            return '' if description.blank?
+
+            desc = description.to_s.strip.tap { |desc| desc[0] = desc[0].upcase }
+            desc += "." unless desc =~ /\.\s*\z/
+            desc
+          end
+
+          def uncapitalize(description)
+            return '' if description.blank?
+
+            description.strip.tap { |desc| desc[0] = desc[0].downcase }
+          end
+
+          # TODO: multi nested
+          def type_and_format(type, item_type = nil)
+            result = case type.to_s.downcase
+              when 'array'
+                { 'type' => 'array', 'items' => { 'type' => item_type.presence || 'string' } }
+              else
+                TYPE_INFO[type.to_s]
+            end
+            result ? result.dup.with_indifferent_access : nil
```

Comment:
Isn't the `dup` redundant when chained immediately with `with_indifferent_access`?